### ISSUE TITLE
Missing commit messages in squash commit

### ISSUE THREAD

**Issue by l0b0** [chdsbd/kodiak#622]
[This PR](https://github.com/linz/geospatial-data-lake/pull/366) with three commits and [this Kodiak configuration](https://github.com/linz/geospatial-data-lake/blob/95f19c014fe430ce2c70dde2ed562c8b9a628641/.kodiak.toml)

```
version = 1

[approve]
auto_approve_usernames = ["dependabot"]

[merge]
method = "squash"

[merge.message]
title = "pull_request_title"
include_coauthors = true
include_pull_request_url = true
```

was merged as [this commit](https://github.com/linz/geospatial-data-lake/commit/95f19c014fe430ce2c70dde2ed562c8b9a628641):

```
commit 95f19c014fe430ce2c70dde2ed562c8b9a628641
Author: Victor Engmark <vengmark@linz.govt.nz>
Date:   Wed Mar 10 11:29:20 2021 +1300

    fix: Show correct badges (#366)
    
    https://github.com/linz/geospatial-data-lake/pull/366
```

I would have expected the commit message to include the messages of each commit per the default [`merge.message.body` configuration](https://kodiakhq.com/docs/config-reference#mergemessagebody):

> By default ("github_default"), GitHub combines the titles of a PR's commits to create the body text of a merge commit.

Is something wrong with the configuration or with Kodiak?

**Comment by chdsbd** [comment]
I think the bug is that "include_coauthors" and "include_pull_request_url" are incompatible with "github_default". 

The GitHub API will only generate the default merge body if we don't send them a body value. But "include_coauthors" and "include_pull_request_url" cause us to create and send a value for the merge body.

If you want to use "github_default I think you'll need to unset "include_coauthors" and "include_pull_request_url".

**Comment by chdsbd** [comment]
Looking at the docs, [`merge.message.include_coauthors`](https://kodiakhq.com/docs/config-reference#mergemessageinclude_coauthors) only works when `merge.message.body = "pull_request"`.

I think we should update the code for `include_pull_request_url` to behave the same way and mention that in the docs too.

**Comment by chdsbd** [comment]
This should be fixed now that #631 is deployed. The docs for include_pull_request_url have been updated: https://kodiakhq.com/docs/config-reference#mergemessageinclude_pull_request_url
